metashare/api/jobs.py

```diff
@@ -179,6 +179,10 @@ def commit_changes_from_org(scratch_org, user, desired_changes, commit_message):
         repository = get_repo_info(user, repo_id=repo_id)
         commit = repository.branch(branch).commit
 
+        scratch_org.task.refresh_from_db()
+        scratch_org.task.has_unmerged_commits = True
+        scratch_org.task.save()
```

Comment:
This is a fine first iteration, but I think we'll want this to call a method on the `task` so we can also trigger a WS event.

Reply:
Yeah, I wanted to talk with you about that. Do we currently subscribe to Tasks? What info do you need updated on them? When?

Reply:
@wlonk I already made the changes there that I think are helpful. Let me know what you think.